_episodes/015-twir-339.md

```diff
@@ -3,7 +3,7 @@ title: "This Week in Rust - Issue 339"
 date: 2020-05-19T14:30:00Z
 file: https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-19.mp3
 duration: "07:34"
-length: "7116560"
+length: "5267717"
```

Comment:
Huh, weird, I wonder why this wasn't caught before..

Reply:
Not sure myself, I was quite surprised when that came up.